SPECS/syslog-ng/syslog-ng.spec

```diff
@@ -167,6 +159,9 @@ fi
 %{_libdir}/pkgconfig/*
 
 %changelog
+* Thu Mar 01 2024 Henry Li <lihl@microsoft.com> - 4.3.1-2
```

Comment:
please update the release number above as well

Reply:
Thanks for capturing this. Fixed!